### PR TITLE
[IMP] account: Import First Bill Modal Improvement

### PR DESCRIPTION
**Pull request by jbw-odoo**
Improve the import your first bill Usability + look & feel.

Task: 2581377

### CODE REVIEW

File: addons/account/wizard/account_tour_upload_bill.py [hunk 1]
```diff
@@ -55,22 +67,28 @@ def _action_list_view_bill(self, bill_ids=[]):
...
             return purchase_journal.with_context(default_journal_id=purchase_journal.id, default_move_type='in_invoice').create_invoice_from_attachment(attachment_ids=self.attachment_ids.ids)
         elif self.selection == 'sample':
+            context = dict(self._context)
             attachment = self.env['ir.attachment'].create({
                 'type': 'binary',
-                'name': 'INV/2020/0011.pdf',
+                'name': 'INV/%s0001' % invoice_date.strftime('%Y/%m'),
                 'res_model': 'mail.compose.message',
-                'datas': self.sample_bill_preview,
+                'datas': self.preview_invoice,
```
Comment (jbw-odoo) [inline addons/account/wizard/account_tour_upload_bill.py:80]: Datas can not be HTML. This must be a file's binary. Ideally a pdf created from the HTML.

Comment (qdp-odoo) [reply addons/account/wizard/account_tour_upload_bill.py:80]: tip: probably using self.env['ir.actions.report']._run_wkhtmltopdf

**Review by qdp-odoo — COMMENTED** [review]
almost there

**Comment by qdp-odoo** [comment]
I fixed the attachment, which is now valid... but there's a css issue that I don't know yet how to solve.

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo retry

**Pull request closed without merging**